.devcontainer/postCreate.sh

```diff
@@ -0,0 +1,15 @@
+#!/bin/sh
+
+# Install dev dependencies from OPAM
+opam init -y --bare --disable-sandboxing git+https://github.com/rescript-lang/opam-repository
```

Comment:
```suggestion
opam init -y --bare --disable-sandboxing
```

Our custom fork of the OPAM repo is not needed anymore, see #6967.

Reply:
Nice, this speeds up things significantly on my end.